### PR TITLE
Fix #13: Feature/setup react

### LINKED ISSUE #13: Feature request: Set up a blank React project
As a developer,
I want to set up a blank React project
So others can work on it

**Priority**
High

**Is your feature request related to a problem? Please describe.**
n/a

**Describe the solution you'd like**
A blank React project is served when accessing 127.0.0.1:1234.

**Describe alternatives you've considered**
n/a

**Issue Dependencies**
No

**Additional context**
n/a

### LINKED ISSUE #14: Feature request: App design
As a front-end developer
I want to have a mockup of the app
So I can build it

**Priority**
High   

**Is your feature request related to a problem? Please describe.**
n/a

**Describe the solution you'd like**
A clean concise mockup of all necessary screens of the app with any necessary notes for devs. The screens can be found [here](https://github.com/SOFTENG701G1/A1/wiki/Mockups).

**Describe alternatives you've considered**
n/a

**Issue Dependencies**
n/a

**Additional context**
It has been decided to use [material library](https://material-ui.com/) for SPLiT UI, so the design should take this into account.

### PR DESCRIPTION
**Pull request by pabloSushibar**
Fix #13 . 

Added boiler plate React-Redux for our project.

### CODE REVIEW

File: server/src/server.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def default(self, *args, **kwargs):
 
     @cherrypy.expose
     def index(self):
-        page = open(SITE_ROOT + "/test/index.html", "r")
+        page = open(SITE_ROOT + "/public/index.html", "r")
```
Comment (williamlixu) [inline server/src/server.py:25]: Not sure how CherryPi -> React will actually work but we can figure this out once we start working on the API

**Review by williamlixu — APPROVED** [review]
Looks good to me! We will have to clean up some of the less relevant things later like the CSS/icons that we don't end up using.

File: split-webapp/split/src/App.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
+.App {
```
Comment (vee-christella) [inline split-webapp/split/src/App.css:1]: Is anything in this file going to remain once we do get started?

**Review by vee-christella — APPROVED** [review]
Looks good to me! Just make sure that we remove all the temporary files/components that we won't use for the real app.

**Pull request merged**